src/main/py/commission/artwork.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+"""
+Module to manage artwork commission class
+
+Artwork class allows us to create a commission, generate a file descriptor
+"""
+
+
+class Artwork:
+    """
+    Class to manage artwork commissions
+    """
+
+    def __init__(self, width, height, wait_time):
+        """
+        Initializes an instance of the Artwork class.
+        - width (float): The width of the artwork.
```

Comment:
It's always helpful to know what the unit is (e.g., cm, inches, pixels). We probably  want pixels in this case.

As far as specifying the type, I would just use the Python type hint feature in the signature, even though we're not running a type checker at this point. The other option, given we're not type checking would be to just not specify anything at this point.